lib/open_project_scimitar.rb

```diff
@@ -30,20 +30,11 @@
 
 module OpenProjectScimitar
   AUTHENTICATION_SCHEMES = [
-    Scimitar::AuthenticationScheme.new(
-      type: "oauth2",
-      name: "OAuth2",
-      description: OpenProject::Static::Links.url_for(:sysadmin_docs, :scim_static_access_token_authentication_method)
-    ),
     Scimitar::AuthenticationScheme.new(
       type: "oauthbearertoken",
       name: "OAuth Bearer Token",
-      description: OpenProject::Static::Links.url_for(:sysadmin_docs, :scim_oauth2_client_credentials_authentication_method)
+      description: "Bearer Token can be obtains in 3 different ways" \
+                   "(#{OpenProject::Static::Links.url_for(:sysadmin_docs, :scim_authentication)})"
```

Comment:
I just now realize that we've always included the URL to our docs in the `description`. Is the `documentationUri` not available through Scimitar?